README.md

```diff
@@ -1,14 +1,19 @@
 # Montly Status email
 
-Apps allowing to send monthly status emails to cloud customers. This is a internal apps.
+Send monthly status mails to users. This app doesn't provide a user interface.
+
+Per default it sends a summary of used storage along with some usage hints.
+
+The default messages are in German.
 
 ## Editing the messages sent
 
-All the messages sent to the users are located inside `lib/Service/MessageProvider.php`.
-To overwrite then for internal usage, you need to create a new class inheriting from the
-MessageProvider and then overwrite any methods with your own messages.
+The messages sent to users are defined at `lib/Service/MessageProvider.php`.
+
+To overwrite the default messages, create a new class inheriting from `MessageProvider`
+and overwrite the desired methods.
 
-You can then edit your config.php to tell this app to use your MessageProvider instead.
+Then configure your `MessageProvider` in `config.php`:
```

Comment:
Maybe would be good to replace by the occ command. What you think about?

```
occ config:system:set status-email-message-provider --value "\MyCustomApp\MyMessageProvider"
```

Reply:
Not sure, people might prefer to edit the `config.php` manually, no?